data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -306,6 +329,15 @@
     "tract_values = (\n",
     "    merged_df[ExtractTransformLoad.GEOID_TRACT_FIELD_NAME].str.len().unique()\n",
     ")\n",
+    "# Debugging: Uncomment lines 24-29 to identify which portions of the dataset\n",
+    "# are not coherent with the scoring comparison's contract\n",
+    "# mask = (merged_df[\n",
+    "# ExtractTransformLoad.GEOID_TRACT_FIELD_NAME].str.len() != 11)\n",
+    "# merged_df = merged_df.loc[mask] \n",
+    "# print(merged_df)\n",
+    "\n",
+    "\n",
```

Comment:
I added these commented-out lines in the event someone needs to identify which tracts violate the length.